openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2097,7 +2096,7 @@ The following is a non-normative example of an object comprising the `credential
 The following additional claims are defined for authorization details of type `openid_credential` and this Credential format.
 
 * `vct`: REQUIRED. String as defined in (#server-metadata-sd-jwt-vc). This claim contains the type values the Wallet requests authorization for at the Credential Issuer. It MUST only be present if the `format` claim is present. It MUST not be present otherwise.
-* `claims`: OPTIONAL. An object as defined in (#server-metadata-sd-jwt-vc).
+* `claims`: OPTIONAL. An object as defined in (#server-metadata-sd-jwt-vc) excluding the `display` parameter.
```

Comment:
As we're only excluding `display`, did we conclude that the `mandatory` and `value_type` do have meaning if passed in the authorization details?

Reply:
```suggestion
* `claims`: OPTIONAL. An object as defined in (#server-metadata-sd-jwt-vc) excluding the `display` and `value_type` parameters. `mandatory` parameter here is used by the Wallet to indicate to the Issuer that it only accepts Credential(s) issued with those claim(s).
```